apps/videos-admin/src/app/(dashboard)/videos/[videoId]/_VideoView/VideoChildren/VideoChildren.tsx

```
@@ -52,105 +35,13 @@
-          sx={{ '& .MuiDialog-paperFullWidth': { maxWidth: 480 } }}
-        >
-          <VideoCreateForm
-            parentId={videoId}
-            close={() => setShowCreateDialog(false)}
-            onCreateSuccess={handleCreateSuccess}
-          />
-        </Dialog>
-      </Section>
-    )
+  if (childVideos.length === 0) {
+    return <Typography>No children to show</Typography>
```

ESLint

[error] disallow literal string: <Typography>No children to show</Typography>

(i18next/no-literal-string)